src/serverlogging.cpp

```diff
@@ -168,9 +168,6 @@ QString CServerLogging::CurTimeDatetoLogString()
     // time and date to string conversion
     const QDateTime curDateTime = QDateTime::currentDateTime();
 
-    // format date and time output according to "3.9.2006, 11:38:08"
-    return QString().setNum ( curDateTime.date().day() ) + "." +
-        QString().setNum ( curDateTime.date().month() ) + "." +
-        QString().setNum ( curDateTime.date().year() ) + ", " +
-        curDateTime.time().toString();
+    // format date and time output according to "2006-09-30 11:38:08"
```

Comment:
This function: https://github.com/corrados/jamulus/blob/master/src/serverlogging.cpp#L103
will not work anymore with your change.